lua/astronvim/autocmds.lua

```diff
@@ -243,11 +243,12 @@ if is_available "neo-tree.nvim" then
     end,
   })
   autocmd("TermClose", {
-    pattern = "*lazygit",
-    desc = "Refresh Neo-Tree git when closing lazygit",
-    group = augroup("neotree_git_refresh", { clear = true }),
+    pattern = "*lazygit*",
+    desc = "Refresh Neo-Tree when closing lazygit",
+    group = augroup("neotree_refresh", { clear = true }),
     callback = function()
       if package.loaded["neo-tree.sources.git_status"] then require("neo-tree.sources.git_status").refresh() end
+      if package.loaded["neo-tree.sources.manager"] then require("neo-tree.sources.manager").refresh("filesystem") end
```

Comment:
```suggestion
      local manager_avail, manager = pcall(require, "neo-tree.sources.manager")
      if manager_avail then
        for _, source in ipairs { "filesystem", "git_status" } do
          local module = "neo-tree.sources." .. source
          if package.loaded[module] then manager.refresh(require(module).name) end
        end
      end
```

I think this is a good point to generalize this a bit and also give us safety against names. This also fixed the module checking for the filesystem source to actually check if the source is loaded rather than just checking if the manager is there. (Forgive me if the code is not formatted correctly or has syntax error, Im typing this on my phone with the GitHub app 😅)